internal/rest/avi_obj_vsvip.go

```diff
@@ -504,13 +504,13 @@ func (rest *RestOperations) AviVsVipCacheDel(rest_op *utils.RestOp, vsKey avicac
 func networkNamesToVips(networkNames []string) []*avimodels.Vip {
 	var vipList []*avimodels.Vip
 	autoAllocate := true
-	for vipIDInt, networkName := range networkNames {
+	for vipIDInt := range networkNames {
```

Comment:
we can retain the range and use the util function `proto.String`

Reply:
Done